include/service/entities/section_service.js

```diff
@@ -462,7 +462,7 @@ module.exports = function SectionServiceModule(pb) {
             //validate for each type of nav item
             switch(navItem.type) {
             case 'container':
-                onDone(null, errors);
+                onDone(null, []);
```

Comment:
@cchatham This fixes the out of memory error.  When ```onDone``` is called it combines the errors in the ```errors``` variable with those where are passed into ```onDone``` as the second parameter.  It was adding errors to itself creating an infinite loop.